src/resources/styles/settings.scss

```
@@ -245,6 +245,9 @@ $login-align-items: center;
 $login-button-focus-border-radius: 4px;
 $login-icon-transform: none;
 $login-button-width: auto;
+// The login card while the encryption explainer is open — the design's 700px
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)